projects/admin/src/app/menu/menu.component.ts

```diff
@@ -176,10 +177,15 @@ export class MenuComponent implements OnInit {
     this.linksMenu = this._menuService.linksMenu;
 
     this._localeStorageService.onSet$.subscribe(() => {
-      const link = this.linksMenu.entries[3].entries.find(
+      const link = this.linksMenu.entries[4].entries.find(
```

Comment:
I do not understand why this is done twice in two different components.

Reply:
your are right. it's useless. removed.